7/root/usr/bin/run-valkey

```diff
@@ -2,16 +2,16 @@
 
 export_vars=$(cgroup-limits); export $export_vars
 source ${CONTAINER_SCRIPTS_PATH}/common.sh
-set -eu
+set -eux
```

Comment:
Was this used only for debugging the initial version or do we want to keep it in the future release as well?

Reply:
Ah I see it gets removed in a follow up commit. For future, in the initial review it would be better if such changes were not present at all in this commit.